Mathlib/NumberTheory/Padics/PadicIntegers.lean

```diff
@@ -192,11 +192,10 @@ instance : Norm ℤ_[p] := ⟨fun z => ‖(z : ℚ_[p])‖⟩
 variable {p} in
 theorem norm_def {z : ℤ_[p]} : ‖z‖ = ‖(z : ℚ_[p])‖ := rfl
 
-instance : NormedCommRing ℤ_[p] :=
-  { PadicInt.instCommRing with
-    dist_eq := fun ⟨_, _⟩ ⟨_, _⟩ => rfl
-    norm_mul := by simp [norm_def]
-    norm := norm }
+instance : NormedCommRing ℤ_[p] where
+  __ := (_ : CommRing ℤ_[p])
```

Comment:
```suggestion
  __ := PadicInt.instCommRing
```
I don't think `_` is guaranteed to do anything useful here.

(also, this seems kind of out of scope for this change)

Reply:
Well, it avoids having to hard-code the name of the earlier instance; I thought it was considered better style to avoid explicitly referring to the names of instances where possible. I've seen style like
```
__ : CommRing ℤ_[p] := inferInstance
```
or 
```
__ := inferInstanceAs (CommRing ℤ_[p])
```
elsewhere in the codebase.